inc/cleantalk-public-integrations.php

```diff
@@ -3401,9 +3401,31 @@ function apbct_form__uwp_validate($result, $_type, $data)
      * Filter for POST
      */
     $input_array = apply_filters('apbct__filter_post', $_POST);
+
+    $long_email = '';
+    foreach ($input_array as $value) {
+        if (preg_match("/^\S+@\S+\.\S+$/", $value) &&
+            strlen($value) > strlen($long_email)
+        ) {
+            $long_email = $value;
+        }
+    }
+
     $data = ct_gfa($input_array);
```

Comment:
Pass `$long_email` into `ct_gfa()` by second parameter. And you will not need to calculate `$sender_email` below.